src/plugins/discourse/mirror.test.js

```diff
@@ -132,6 +164,20 @@ class MockFetcher implements Discourse {
     };
   }
 
+  addCategory(topicId: TopicId): PostId {
+    const postId = this.addPost(topicId, null);
+    this._topicCategories.set(topicId, topicId);
```

Comment:
The topicCategories is supposed to map to categoryIds...why does it get set to a topicId here?